File: problems/problems_3375/Solution.java
package problems.problems_3375;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int minOperations(int[] nums, int k) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] nums = jsonArrayToIntArray(inputJsonValues[0]);
		int k = Integer.parseInt(inputJsonValues[1]);
        return JSON.toJSON(minOperations(nums, k));
    }
}

File: problems/problems_3375/solution.py
import solution
from typing import *


class Solution(solution.Solution):
    def solve(self, test_input=None):
        return self.minOperations(*test_input)

    def minOperations(self, nums: List[int], k: int) -> int:
        ans, m = set(), nums[0]
        for num in nums:
            ans.add(num)
            m = min(m, num)
        if k > m:
            return -1
        return len(ans) - 1 if k in ans else len(ans)
